src/set_theory/schroeder_bernstein.lean

```diff
@@ -19,24 +19,23 @@ namespace embedding
 section antisymm
 variables {α : Type u} {β : Type v}
 
-theorem schroeder_bernstein {f : α → β} {g : β → α}
-  (hf : function.injective f) (hg : function.injective g) : ∃h:α→β, bijective h :=
-let s : set α := lfp $ λs, (g '' (f '' s)ᶜ)ᶜ in
-have hs : s = (g '' (f '' s)ᶜ)ᶜ,
-  from lfp_eq $ assume s t h,
-    compl_subset_compl.mpr $ image_subset _ $
-    compl_subset_compl.mpr $ image_subset _ h,
+theorem schroeder_bernstein {f : α  →  β} {g : β  →  α}
```

Comment:
Any reason for the extra spaces around the `→` here and below?

Reply:
Oops no